recipe/install_nvcc.sh

```diff
@@ -53,17 +63,33 @@ then
     return 1
 fi
 
-if [[ \$(grep -q "CUDA Version ${PKG_VERSION}" \${CUDA_HOME}/version.txt) -ne 0 ]]
+if [[ -z "\$(\${CUDA_HOME}/bin/nvcc --version | grep "Cuda compilation tools, release ${PKG_VERSION}")" ]]
 then
-    echo "Version of installed CUDA didn't match package"
-    return 1
+  echo "Version of installed CUDA didn't match package"
+  return 1
 fi
 
 export CUDA_HOME="\${CUDA_HOME}"
 export CFLAGS="\${CFLAGS} -I\${CUDA_HOME}/include"
 export CPPFLAGS="\${CPPFLAGS} -I\${CUDA_HOME}/include"
 export CXXFLAGS="\${CXXFLAGS} -I\${CUDA_HOME}/include"
 
+### CMake configurations
+
+# CMake looks up components in CUDA_PATH, not CUDA_HOME
+export CUDA_PATH="\${CUDA_HOME}"
+# New-style CUDA integrations in CMake
+CMAKE_ARGS="\${CMAKE_ARGS:-} -DCUDAToolkit_ROOT=\${CUDA_HOME}"
+# Old-style CUDA integrations in CMake
+## See https://github.com/conda-forge/nvcc-feedstock/pull/58#issuecomment-752179349
+CMAKE_ARGS+=" -DCUDA_TOOLKIT_ROOT_DIR=\${CUDA_HOME}"
+## Avoid https://github.com/conda-forge/openmm-feedstock/pull/44#issuecomment-753560234
+## We need CUDA_HOME in _front_ of CMAKE_FIND_ROOT_PATH
+CMAKE_ARGS="\$(echo \${CMAKE_ARGS} | sed -E -e "s|(-DCMAKE_FIND_ROOT_PATH=)(\S+)|\1\$CUDA_HOME;\2|")"
```

Comment:
Can you add sed to `run` requirements?

Reply:
This appears to be done below. Marking as resolved.